components/script/fetch.rs

```diff
@@ -235,6 +242,18 @@ impl FetchResponseListener for FetchContext {
         // TODO
         // ... trailerObject is not supported in Servo yet.
     }
+
+    fn resource_timing(&mut self) -> &mut ResourceFetchTiming {
+        &mut self.resource_timing
+    }
+
+    fn submit_resource_timing(&self) {
+        let local_name = DOMString::from("fetch");
+        let global = self.global.root();
+        let performance_entry = PerformanceResourceTiming::new(
+            &global, global.get_url().clone(), local_name, None, &self.resource_timing);
```

Comment:
We should store the url from RequestInit in FetchContext instead of using the global's URL here.